src/neo/Wallets/Wallet.cs

```diff
@@ -435,8 +458,28 @@ public bool Sign(ContractParametersContext context)
                         byte[] signature = context.Verifiable.Sign(key);
                         fSuccess |= context.AddSignature(account.Contract, key.PublicKey, signature);
                     }
+                    else
```

Comment:
If `account` is `null`, we can try contract verification.

Reply:
Done